src/modules/auth/components/TermsComponent.tsx

```diff
@@ -0,0 +1,11 @@
+export function TermsComponent(): JSX.Element {
```

Comment:
Have in mind, that if we have a component for this, we should have a way of checking if the user checked the checkbox, since if he didn't, we won't let him submit a form and continue with registration.